### PR TITLE
PWGHF: check on number of process function enabled toXiPi task

### PR DESCRIPTION
**Pull request by ZFederica**
Implement a check in the init function of the candidateCreatorToXiPi to make sure that only one process function is enabled (either do the indexes combinatorics or run on derived data from trackIndexSkimCreator)

### CODE REVIEW

File: PWGHF/TableProducer/candidateCreatorToXiPi.cxx [hunk 1]
```diff
@@ -110,6 +110,10 @@ struct HfCandidateCreatorToXiPi {
     ccdb->setLocalObjectValidityChecking();
     lut = o2::base::MatLayerCylSet::rectifyPtrFromFile(ccdb->get<o2::base::MatLayerCylSet>(ccdbPathLut));
     runNumber = 0;
+
+    if (doprocessIdxCombinatorics == true && doprocessDerivedData == true) {
```
Comment (fgrosa) [inline PWGHF/TableProducer/candidateCreatorToXiPi.cxx:114]:
```suggestion
    if (doprocessIdxCombinatorics && doprocessDerivedData) {
```

**Review by fgrosa — CHANGES_REQUESTED** [review]
Ciao @ZFederica I just have a small comment. In the future, you can consider to remove completely the process function that computes the combinatorics, since you can in principle always use the `trackIndexSkimCreator` task to do it and then you avoid to have two equivalent tasks to maintain

File: PWGHF/TableProducer/candidateCreatorToXiPi.cxx (hunk 1 above)
Comment (vkucera) [reply PWGHF/TableProducer/candidateCreatorToXiPi.cxx:114]: One should also check that they are not both `false`.

**Comment by ZFederica** [comment]
@fgrosa yes I will remove the first process, I would just keep it for a little bit longer for validation

**Review by fgrosa — APPROVED** [review]

**Pull request merged**